src/cmd/singularity/cli/sif.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/singularityware/singularity/src/pkg/buildcfg"
+	"github.com/singularityware/singularity/src/pkg/sylog"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	sifCmd.Flags().SetInterspersed(false)
+	singularityCmd.AddCommand(sifCmd)
+}
+
+var sifCmd = &cobra.Command{
+	Use:    "sif COMMAND OPTION FILE",
+	Hidden: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		var sif = buildcfg.SBINDIR + "/sif"
+
+		sifCmd := exec.Command(sif, args...)
+		sifCmd.Stdout = os.Stdout
+		sifCmd.Stderr = os.Stderr
+
+		if err := sifCmd.Start(); err != nil {
+			sylog.Fatalf("%v", err)
+		}
+		if err := sifCmd.Wait(); err != nil {
+			sylog.Errorf("%v", err)
```

Comment:
Sorry @ArangoGutierrez, I missed this one on my first pass, think that should be `Fatalf`. If `sif` returns a failure, `singularity sif` should also.